backend/endpoints/volunteer.py

```diff
@@ -181,7 +180,7 @@ def get_volunteers_by_filters(filters, pages=0, per_page=10000):
         if len(filters) > 0:
             flt = {}
             toBool = {"true": True, "false": False}
-            caseS = ["first_name", "last_name"]
+            caseS = ["first_name", "last_name", "phone", "zone", "role"]
```

Comment:
Sorry, there is not enough information in the job description. Volunteers should be filtered either by first name, last name, or by phone. At the same time, they can be additionally filtered by zone and role. Here is  the [design](https://www.figma.com/proto/oCN3NSECKQnS4PWhZnWGCT/Dashboard_Voluntar.md_V2.0?node-id=585%3A4930&viewport=402%2C967%2C0.4029495418071747&scaling=scale-down-width)